shop/src/main/java/ro/msg/learning/shop/dto/StockGetDto.java

```diff
@@ -0,0 +1,19 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.domain.Location;
+import ro.msg.learning.shop.domain.Product;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class StockGetDto {
+
+    private Product product;
```

Comment:
a dto cannot have an entity inside it. 
the entity is a direct representation of the db and creating a a dependency between them might lead to later on problems.
also, we use layer architecture with the purpose of separating the layers of interest and mainting a separation of concern. This refers also to dependencies. 
https://www.linkedin.com/pulse/difference-between-entity-dto-what-use-instead-omar-ismail/

Reply:
also, this StockGetDto is awfully similar to StockCreateDto. Why are you using both?
using just one dto, such as StockDto could simplify things.
Also, very funny how the same logic of creating these typoes of DTOs can be seen in someone else's code 👀 
Magic rules:
1. if you intend to inspire yourself from ANYWHERE, please check with your logic, take your decision, make "inspired" code better than you found it. 
2. if you inspire yourself from another person which i am also reviewing, make rule number 1 a priority 😆

Reply:
Done. Now I have a single Dto.